gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/PageMapper.java

```diff
@@ -100,6 +102,14 @@ public interface PageMapper {
     @Mapping(target = "uploadDate", source = "createdAt")
     MediaEntity mapMedia(Media media);
 
+    default String map(SourceConfigurationConfiguration value) {
+        try {
+            return mapper.writeValueAsString(value);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException("Failed to serialize configuration", e);
```

Comment:
Instead of a generic RuntimeException, let's throw a `TechnicalManagementException` (like in line 170)

Reply:
Done